src/app/components/Parameters/Parameters.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import React, { ChangeEvent } from 'react';
+import { useParametersContext } from '@/context/parameters.context';
+import styles from './Parameters.module.scss';
+
+function Parameters() {
+  const { selects, setSelects } = useParametersContext();
+
+  const handleChange = (event: ChangeEvent<HTMLSelectElement>) => {
+    const { name, value } = event.target;
+    setSelects({ ...selects, [name]: value });
+  };
+
+  return (
+    <form className={styles.parameters}>
+      <label htmlFor="sortBy">
+        Sort by:
+        <select name="sortBy" onChange={handleChange} defaultValue="">
```

Comment:
Please create a `Select` component and pass there options and default option props